packages/gatsby-plugin-typescript/src/gatsby-node.js

```diff
@@ -21,16 +21,25 @@ function onCreateWebpackConfig({ actions, loaders }) {
     return
   }
 
+  const { version } = require(`gatsby/package.json`)
+  const [major, minor] = version.split(`.`).map(Number)
+  const doesUsedGatsbyVersionSupportResourceQuery =
+    (major === 4 && minor >= 19) || major > 4
```

Comment:
let's try to handle case when require resolution won't work - not ideal, but probably better than crashing
```suggestion
  let doesUsedGatsbyVersionSupportResourceQuery
  try {
    const { version } = require(`gatsby/package.json`)
    const [major, minor] = version.split(`.`).map(Number)
    doesUsedGatsbyVersionSupportResourceQuery =
    (major === 4 && minor >= 19) || major > 4
  } catch {
    doesUsedGatsbyVersionSupportResourceQuery = true
  }
```